src/data/ws-area_registry.ts

```diff
@@ -1,19 +1,12 @@
 import { Connection, createCollection } from "home-assistant-js-websocket";
 import { Store } from "home-assistant-js-websocket/dist/store";
-import { stringCompare } from "../common/string/compare";
-import { AreaRegistryEntry } from "./area_registry";
 import { debounce } from "../common/util/debounce";
+import { AreaRegistryEntry } from "./area_registry";
 
 const fetchAreaRegistry = (conn: Connection) =>
-  conn
-    .sendMessagePromise({
-      type: "config/area_registry/list",
-    })
-    .then((areas) =>
-      (areas as AreaRegistryEntry[]).sort((ent1, ent2) =>
-        stringCompare(ent1.name, ent2.name)
-      )
-    );
```

Comment:
@bramkragten Is there a particular reason why `sort` was removed here? The areas dashboard and areas target picker are all in random order now. Should alphabetical sort be readded in the consumer of this function, or what was the thought here?